src/codewhisperer/tracker/codewhispererCodeCoverageTracker.ts

```diff
@@ -24,6 +24,8 @@ interface CodeWhispererToken {
     accepted: number
 }
 
+const autoClosingKeystrokeInputs = ['[]', '{}', '()', `\"\"`, `\'\'`]
```

Comment:
what is the intention of `\` ? do you want literal `"` or a literal `\"` ? If you only want literal `"` (and `'`), this should work:
```suggestion
const autoClosingKeystrokeInputs = ['[]', '{}', '()', '""', "''"]
```